modules/ossm-security-mtls.adoc

```diff
@@ -23,10 +23,26 @@ kind: ServiceMeshControlPlane
 spec:
   version: v2.0
   security:
-    controlPlane:
+    dataPlane:
       mtls: true
 ----
 
+You can also enable mTLS by using the {product-title} web console.
+
+.Procedure
+
+. Log in to the web console.
+
+. Click the *Project* menu and choose the `istio-system` project from the list.
+
+. Click *Operators* -> *Installed Operators*.
+
+. Click on *Service Mesh Control Plane* under *Provided APIs*.
+
+. Click the name of your `ServiceMeshControlPlane` resource, for example, `production`.
+
+. On the Details page, click the toggle in the *Security* section for *Data Plane Security*.
```

Comment:
Since the form view is automatically generated, I've been documenting using the YAML view for configuration in the console.  It also makes more sense to me if we're giving YAML examples to use the YAML view.   I'm also not 100% sure that changes made in the Details view would be reloaded?  When I tried this in a cluster and set the mTLS to true, the "AppliedSpec" still read "False"  (meaning, I'm not sure the OpenShift GUI performs a "reload" function).

Reply:
For me, when I test it again, it works. If the UI doesn't update the resource when you do it, can you open a bug against engineering or bring it up with QE? 

I think adding content about how to do it in the UI is an important part of the PR. Our customers asked about the toggles in production UI. The toggles make this work a lot easier than working with the YAML.

Reply:
It looks like there's just a bit of a time lag.